schemas/2019-08-01-preview/Microsoft.OperationalInsights.json

```diff
@@ -46,21 +56,21 @@
               "$ref": "#/definitions/Identity"
             },
             {
-              "$ref": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+              "$ref": "https://schema.management.azure.com/schemas/common/definitions.json#/definitions/expression"
             }
           ],
-          "description": "The identity of the resource."
+          "description": "Identity for the resource."
         },
         "properties": {
           "oneOf": [
             {
               "$ref": "#/definitions/ClusterProperties"
             },
             {
-              "$ref": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+              "$ref": "https://schema.management.azure.com/schemas/common/definitions.json#/definitions/expression"
```

Comment:
Just to clarify for me: here referenced common/definitions.json
Is schemas/common/autogeneratedResources.json referenced from common/definitions.json?

Reply:
Sure, this change is really just a restructuring - we wanted to avoid the circular dependency that we currently have (`2015-01-01/deploymentTemplate.json` -> `{individual resource}.json` -> `2015-01-01/deploymentTemplate.json`). 

The actual definition this points to is exactly the same, the dependency order is just updated to (`2019-04-01/deploymentTemplate.json` -> `common/autogeneratedResources.json` -> `{individual resource}.json` -> `common/definitions.json`)